tests/sklearn/test_sklearn_autolog.py

```diff
@@ -121,8 +121,9 @@ def _get_model_uri(name: str = MODEL_DIR) -> str:
     """
     Get the model URI for the last active run.
     """
-    last_active_run = mlflow.last_active_run()
-    return f"runs:/{last_active_run.info.run_id}/{name}"
+    return next(
+        m.model_uri for m in mlflow.search_logged_models(output_format="list") if m.name == name
+    )
```

Comment:
For a search estimator, a single fit call logs two logged models, so `last_logged_model` can't load the first model.